packages/components/package.json

```diff
@@ -19,9 +19,6 @@
     "dist/*"
   ],
   "dependencies": {
-    "@jobber/design": "^0.55.1",
```

Comment:
I assume this broke the linting but I could be wrong

Reply:
<img width="1378" alt="image" src="https://github.com/GetJobber/atlantis/assets/17303106/e87b47cc-ecf8-4843-8e66-61bebe1d2e5a">

Reply:
Woops, that's a change for another branch. Good catch Michael, gimme 30 seconds to revert those.